ci-operator/step-registry/openstack/deprovision/bastionproxy/openstack-deprovision-bastionproxy-commands.sh

```diff
@@ -11,6 +11,30 @@ fi
 
 export OS_CLIENT_CONFIG_FILE=${SHARED_DIR}/clouds.yaml
 CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
+BASTION_FIP=$(<"${SHARED_DIR}"/BASTION_FIP)
+
+# configure the local container environment to have the correct SSH configuration
+if ! whoami &> /dev/null; then
+    if [[ -w /etc/passwd ]]; then
+        echo "${BASTION_USER}:x:$(id -u):0:${BASTION_USER:-centos} user:${HOME}:/sbin/nologin" >> /etc/passwd
+    fi
+fi
+# shellcheck disable=SC2140
+SSH_ARGS="-o ConnectTimeout=10 -o "StrictHostKeyChecking=no" -i ${CLUSTER_PROFILE_DIR}/ssh-privatekey"
+SSH_CMD="ssh ${SSH_ARGS} ${BASTION_USER}@${BASTION_FIP}"
+SCP_CMD="scp ${SSH_ARGS}"
+
+>&2 echo "Collecting squid logs from 'bastionproxy-$CLUSTER_NAME'"
+$SSH_CMD bash - <<EOF
+mkdir -p /tmp/squid-logs
+sudo cp /var/log/squid/access.log /tmp/squid-logs || true
+sudo cp /var/log/squid/cache.log /tmp/squid-logs || true
+sudo chown -R ${BASTION_USER}: /tmp/squid-logs || true
+tar -czC "/tmp" -f "/tmp/squid-logs.tar.gz" squid-logs/
+EOF
+$SCP_CMD ${BASTION_USER}@${BASTION_FIP}:/tmp/squid-logs.tar.gz ${ARTIFACT_DIR}
+
+>&2 echo "Collecting squid logs from 'bastionproxy-$CLUSTER_NAME'"
```

Comment:
Is this a dup of line 27?

Reply:
We still have the duplicated log. Should we remove it now?
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/20672/rehearse-20672-periodic-ci-openshift-release-master-nightly-4.9-e2e-openstack-az/1432634045732229120/artifacts/e2e-openstack-az/openstack-deprovision-bastionproxy/build-log.txt